tutorials/Coordinates/Coordinates.ipynb

```diff
@@ -0,0 +1,781 @@
+     ]
+    },
+    {
+     "cell_type": "markdown",
+     "metadata": {},
+     "source": [
+      "Note that this tutorial requires astropy v1.0 or higher."
+     ]
+    },
+    {
+     "cell_type": "code",
+     "collapsed": false,
+     "input": [
+      "import urllib\n",
+      "import IPython.display\n",
+      "import numpy as np"
+     ],
+     "language": "python",
+     "metadata": {},
+     "outputs": []
+    },
+    {
+     "cell_type": "code",
+     "collapsed": false,
+     "input": [
+      "%matplotlib inline\n",
+      "from matplotlib import pyplot as plt"
+     ],
+     "language": "python",
+     "metadata": {},
+     "outputs": []
+    },
+    {
+     "cell_type": "code",
+     "collapsed": false,
+     "input": [
+      "from astropy import units as u\n",
+      "from astropy.coordinates import SkyCoord\n",
+      "from astropy.table import Table"
+     ],
+     "language": "python",
+     "metadata": {},
+     "outputs": []
+    },
+    {
+     "cell_type": "heading",
+     "level": 2,
+     "metadata": {},
+     "source": [
+      "Usinging Coordinates to describe on-sky locations and match catalogs"
+     ]
+    },
+    {
+     "cell_type": "markdown",
+     "metadata": {},
+     "source": [
+      "Lets start by considering a field around the picturesque Hickson Compact Group 7.  To do anything with this, we need to get an object that represents the coordinates of the center of this group.\n",
+      "\n",
+      "In Astropy, the most common object you'll work with for coordinates is `SkyCoord`.  A `SkyCoord` can be created most easily directly from angles as shown below.  It's also wise to explicitly specify the frame your coordinates are in, although this is not strictly necessary because the default is ICRS. \n",
+      "\n",
+      "(If you're not sure what ICRS is, it's basically safe to think of it as an approximation to an equatorial system at the J2000 equinox)."
+     ]
+    },
+    {
+     "cell_type": "code",
+     "collapsed": false,
+     "input": [
+      "hcg7_center = SkyCoord(9.81625*u.deg, 0.88806*u.deg, frame='icrs')\n",
+      "hcg7_center"
+     ],
+     "language": "python",
+     "metadata": {},
+     "outputs": []
+    },
+    {
+     "cell_type": "markdown",
+     "metadata": {},
+     "source": [
+      "SkyCoord will also accept string-formatted coordinates, although you'll need to give units if it's not obvious from the format."
+     ]
+    },
+    {
+     "cell_type": "code",
+     "collapsed": false,
+     "input": [
+      "SkyCoord('0h39m15.9s', '0d53m17.016s', frame='icrs')"
+     ],
+     "language": "python",
+     "metadata": {},
+     "outputs": []
+    },
+    {
+     "cell_type": "code",
+     "collapsed": false,
+     "input": [
+      "SkyCoord('0:39:15.9', '0:53:17.016', unit=(u.hour, u.deg), frame='icrs')"
+     ],
+     "language": "python",
+     "metadata": {},
+     "outputs": []
+    },
```

Comment:
A few questions that arise for me after looking at the above examples:
1) Can I use angles in degrees but without explicitly creating a quantity like `9.1235 * u.deg`?  Can I also use bare numeric values along with the `unit=` argument like one can with strings?  Ex: `SkyCoord(9.81625, 0.88806, unit=(u.deg, u.deg))`

2) For that matter, why is it "unit" instead of "units" given that I can give different units for each coordinate?  If I give `unit=u.deg` will that mean that both coordinates are in degrees?  Or am I supposed to think of "unit" this case as some sort of product space of units?

3) Instead of using the `u.deg` or `u.hour` singletons, can I just use strings?  Ex: `SkyCoord(9.81625, 0.88806, unit=('deg', 'deg'))`  I know when I use Astropy for other things I'm usually actually lazy about importing the units module until I absolutely have to--I often hope I can get away with using strings for the units and assume they'll be converted to unit objects appropriately.